ws/ws.nim

```diff
@@ -61,7 +61,7 @@ proc connect*(
   ##
 
   var rng = if isNil(rng): newRng() else: rng
-  var key = Base64.encode(genWebSecKey(rng))
+  var key = Base64Pad.encode(genWebSecKey(rng))
```

Comment:
Does it have to be padded, I don't remember seeing this in the spec?

Reply:
it is not mentioned directly, but all `Sec-WebSocket-Key` always include padding char `=`. probably they assume developers already understand what is the meaning of this `=` char.

the autobahn server also reject our non-padded `Sec-WebSocket-Key` with message something like `..ASCII 24 chars...`.
it need fixed length `Sec-WebSocket-Key`

Reply:
Yeah, more like they assume base64 to be always padded on the network, not a bad assumption but without any practical implication wince headers are well delimited and there is no danger of collapsing base64 streams.